docs/userGuide/deployingTheSite.md

```diff
@@ -52,18 +52,44 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 ### Deploying to GitHub Pages via Travis CI
 **You can setup [<tooltip content="a platform for Continuous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
 
-Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+#### Adding your repository to Travis CI
+
+<panel header="{{icon_info}} Travis CI Migration" type="info" expanded>
+
+As of May 2018, Travis CI was [undergoing migration to `travis-ci.com`](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/), which changes the way you setup repositories on Travis CI. 
+
+- If you are new to Travis CI, or adding a new repository, you should **add your repository on `travis-ci.com`**.
+- If you are adding an existing repository, you should **add your repository on `travis-ci.org`**.
+</panel>
+
+<tabs>
+  <tab header="Add your repository on `travis-ci.com`">
+
+1. [Sign in to Travis](https://travis-ci.com/signin) using your GitHub account.
 1. Accept the authorisation for Travis CI when you are redirected to GitHub.
-1. Click the green _Activate_ button, and select the repository with the MarkBind site.
+1. Go to the [Repositories page](https://travis-ci.org/account/repositories), and click on the green _Activate_ button.
+1. Select the repository with the MarkBind site, and add the Travis CI GitHub App to the repository by clicking the green _Approve and Install_ button.
+
+  </tab>
+  <tab header="Add your repository on `travis-ci.org`">
+
+1. [Sign in to Travis](https://travis-ci.com/signin) using your GitHub account.
+1. Accept the authorisation for Travis CI when you are redirected to GitHub.
+1. Go to the [Repositories page](https://travis-ci.org/account/repositories).
+1. Find the repository with the MarkBind site.<br>
+  %%{{ icon_info }} If the organization/repository is not shown in the list, click on `Review and add` link at the bottom of the list of organization and follow the steps to give Travis access to the organization containing your repo. After that, come back to the [Repositories page](https://travis-ci.org/account/repositories) and use the `Sync Account` button to sync your Travis account with GitHub.%%
+1. Activate the repo using the slider switch in front of it.
+  <include src="screenshot.md" boilerplate var-alt="Activate Repo" var-file="travisActivateRepo.png" inline />
+  </tab>
+</tabs>
+
+#### Configuring your repository in Travis CI
+
 1. [Generate a GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token) with **repo** permissions. Take note of the generated token - you will not be able to see it again once you navigate away from the page.
```

Comment:
Should we elaborate on what we mean by `repo permissions`? i.e., specify exactly which box to tick

Reply:
Added a screenshot for this bit.